html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="post">
+      <label for="email">
+        Email
+      </label>
+      <div>
+        <input type="text" name="emailEntry" id="email">
+      </div>
+      <label for="codingForm">
+        Why is coding cool?
+      </label>
+      <div>
+        <select name="coding" id="codingForm">
+          <option>Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Please select your favorite genre</p>
+      <input type="radio" name="language" value="HTML">
```

Comment:
The value of the `id` attribute on the `input` needs to match the value of the `for` attribute on the `label` it corresponds with.